src/main/java/db/migration/V1_0_0__CreateUserTable.java

```diff
@@ -22,5 +21,17 @@ public void migrate(final Context context) throws Exception {
                 + "wallet_number VARCHAR(16)\n"
                 + ");"
         );
+
+        template.execute("CREATE TABLE quote(\n" +
+                "    quote_id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,\n" +
+                "    ticker VARCHAR(5) NOT NULL,\n" +
+                "    open_price decimal(6,2) NOT NULL,\n" +
+                "    close_price decimal(6,2),\n" +
+                "    high_price decimal(6,2),\n" +
+                "    low_price decimal(6,2),\n" +
```

Comment:
every price should be with 4 digits precision - this is current Warsaw Stock Exchange pricing precision